install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -99,66 +104,214 @@ pv0001                   <none>    5368709120   RWO           Available
 ----
 ====
 
-Users can then link:../../dev_guide/persistent_volumes.html[request storage
-using persistent volume claims], which can now utilize your new persistent
-volume.
+The next step can be to create a persistent volume claim (PVC) which will bind
+to the new PV:
 
-[[nfs-enforcing-disk-quotas]]
+.PVC Object Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nfs-claim1
+spec:
+  accessModes:
+    - ReadWriteOnce <1>
+  resources:
+    requests:
+      storage: 1Gi <2>
+----
+<1> As mentioned above for PVs, the `*accessModes*` do not enforce security, but
+rather act as labels to match a PV to a PVC.
+<2> This claim will look for PVs offering *1Gi* or greater capacity.
+====
+
+Save the definition to a file, for example *_nfs-claim.yaml_*, and create the
+PVC:
+
+====
+----
+# oc create -f nfs-claim.yaml
+----
+====
+
+[NOTE]
+====
+PVs and PVCs are not necessary, just convenient, and make sharing a volume
```

Comment:
I disagree with this sentence -- it is far more convenient to do less work, not more, when using a volume (my subjective opinion).  The real motivation for PV framework is to enable modeling the volumes themselves as cluster resources, to mediate access to those resources to avoid conflicts, and to be able to introspect who is using which storage.